packages/lesswrong/components/users/UsersNameDisplay.jsx

```diff
@@ -24,6 +26,11 @@ const styles = theme => ({
     top: 2,
     color: "rgba(255,255,255,.8)"
   },
+  bookIcon: {
+    width: "1rem",
+    height: "1rem",
```

Comment:
why does this have width/height when it's merged with icon via classNames?

Reply:
They're redundant, I'll take them out.